### PR TITLE
Update fingerprint invocation and schema accordingly

### PR DESCRIPTION
**Pull request by JJ-Pineda**
Resolves [#121](https://github.com/rdkit-rs/cheminee/issues/121) by replacing RDK fingerprints with pattern fingerprints. This has been updated in the `descriptor_v1` schema. Additionally, we now have an extra bytes field in the `descriptor_v1` schema for Morgan fingerprints. The CLI and API indexing actions have been updated to automatically index both the pattern and Morgan fingerprints for every molecule.

### CODE REVIEW

File: src/indexing/mod.rs [hunk 1]
```diff
@@ -77,26 +77,27 @@ pub fn open_index(p: impl AsRef<Path>) -> eyre::Result<Index> {
...
+//             } else if let serde_json::Value::Object(obj1_map) = obj1 {
+//                 return Some(serde_json::Value::Object(obj1_map));
+//             }
+//             Some(obj2)
+//         }
+//         (Some(obj1), None) => Some(obj1),
+//         (None, Some(obj2)) => Some(obj2),
+//         (None, None) => None,
+//     }
+// }
```
Comment (xrl) [inline src/indexing/mod.rs:103]: uh oh! gotta delete this 😄

Comment (JJ-Pineda) [reply src/indexing/mod.rs:103]: So I mainly left it as I imagine we might need it once we incorporate similarity search down the road (i.e. to record scaffolds or custom descriptors, etc).

Comment (xrl) [reply src/indexing/mod.rs:103]: I would come back to the code in the git history in the future. Keeps our codebase cleaner having only "live" code and whatever comments we have should be reserved for documentation about the "live" code.

Comment (JJ-Pineda) [reply src/indexing/mod.rs:103]: Fair enough. Will do

**Review by xrl — APPROVED** [review]

**Pull request merged**